blk/components/virt.c

```diff
@@ -175,18 +175,15 @@ static bool handle_mbr_reply()
 
     microkit_arm_vspace_data_invalidate(mbr_req_data.drv_addr,
                                         mbr_req_data.drv_addr + (BLK_TRANSFER_SIZE * mbr_req_data.count));
-    memcpy(&msdos_mbr, (void *)mbr_req_data.drv_addr, sizeof(struct msdos_mbr));
+    sddf_memcpy(&msdos_mbr, (void *)mbr_req_data.drv_addr, sizeof(struct msdos_mbr));
```

Comment:
Better to use the builtin version if we can.

Reply:
I think the issue here was that the blk libvmm example is built with clang, which doesn't have a built in. For the moment we were using our own string.h, but there is a student currently working on creating a libc for us in the future, so this is a temporary solution (Double check this with @Ivan-Velickovic )